charts/consul/templates/api-gateway-controller-deployment.yaml

```diff
@@ -59,10 +59,6 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: status.hostIP
-        - name: IP
```

Comment:
@nickethier @andrewstucki  can you just double check this. I'm removing this env var because I think it was only used for `-sds-server-host`, just wanted to make sure it's not read inside the `consul-api-gateway server` command.

Reply:
Yep, it should be unused at this point.